### PR TITLE
fixed share screen issue

### PR DESCRIPTION
**Pull request by Yeghro**

### CODE REVIEW

**Review by bitkarrot — COMMENTED** [review]
is 'ha button' needed in any context? or is it just the from the old codebase?

File: public/js/RoomClient.js [hunk 1]
```diff
@@ -2531,9 +2530,6 @@ class RoomClient {
                 pn = document.createElement('button');
                 pn.id = id + '__pin';
                 pn.className = html.pin;
-                ha = document.createElement('button');
```
Comment (bitkarrot) [inline public/js/RoomClient.js:2534]: hideAll button - line 2534 RoomClient.js

**Comment by bitkarrot** [comment]
Merging and pushing to Staging

File: public/js/RoomClient.js (hunk 1 above)
Comment (Yeghro) [reply public/js/RoomClient.js:2534]: likely old code.  I don't recall needing it for anything.

**Pull request merged**